app/questionnaire/questionnaire_schema.py

```diff
@@ -361,6 +380,17 @@ def _populate_answer_dependencies(self) -> None:
                                 option["detail_answer"], block_id=block["id"]
                             )
 
+    def _update_dependencies_for_repeating_blocks(self, block: ImmutableDict) -> None:
```

Comment:
Worth updating name to `_update_dependencies_for_list_repeating_blocks`. Applies to all references of repeating blocks.

Reply:
updated all except the ones in the list collector classes themself 👍